tools/mempool-pusher.sh

```diff
@@ -162,7 +168,10 @@ usage() {
    log "Usage: $0 MODE [ARGS...]"
    log "MODE can be the following:"
    log "      $0 daemon PATH_TO_MEMPOOL LOCAL_PEER_HOST:PORT"
+   log "      $0 daemon PATH_TO_MEMPOOL LOCAL_PEER_HOST1:PORT,LOCAL_PEER_HOST2:PORT"
```

Comment:
For the usage section, can it have an ellipsis at the end, so it reads `LOCAL_PEER_HOST1:PORT,LOCAL_PEER_HOST2:PORT,...` to indicate that a list of host:ports will be taken?

Reply:
Added elipsis, agreed that it makes it more clear more than 2 hosts can be in the list